developerportal/apps/home/templates/home.html

```diff
@@ -35,6 +35,11 @@
         {% include "organisms/topic-links.html" with topics=page.primary_topics.all pagetheme='home' %}
       </section>
     {% endif %}
+    {% if directory_pages.people.latest_authors %}
+      <section id="latest-authors" class="section">
+        {% include "organisms/people-section.html" with people=directory_pages.people.latest_authors show_link=True %}
```

Comment:
I’m starting to wonder whether this should be a new, explicitly chosen list of people with it being the home page.

Reply:
Raised #324 to implement this.